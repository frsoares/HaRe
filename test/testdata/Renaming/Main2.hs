-- This is a main module without an explicit module statement.
-- renaming "bar" should succeed, but "main" fail.

main = putStrLn "hello"

bar x = x ^ 2

