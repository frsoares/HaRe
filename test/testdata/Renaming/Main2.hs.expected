-- This is a main module without an explicit module statement.
-- renaming "bar" should succeed, but "main" fail.

main = putStrLn "hello"

baz x = x ^ 2